app/react-native/src/preview/components/OnDeviceUI/index.js

```diff
@@ -1,24 +1,116 @@
-import React, { PropTypes } from 'react';
-import { View } from 'react-native';
+import React, { Component, PropTypes } from 'react';
+import { Animated, Easing, View, TouchableWithoutFeedback, Image, Text } from 'react-native';
 import style from './style';
 import StoryListView from '../StoryListView';
 import StoryView from '../StoryView';
 
-export default function OnDeviceUI(props) {
-  const { stories, events, url } = props;
+export default class OnDeviceUI extends Component {
+  constructor(props, ...args) {
+    super(props, ...args);
```

Comment:
Good catch. I had props in there before and just forgot to clean this up. I'll get it updated.